README.md

```diff
@@ -61,17 +62,19 @@ const num1 = 5;
 const num2 = 10;
 ```
 
+
 ### Your solution here:
+console.log(num1+num2)
 5.  How can we make `num3` equal to the sum of `num1` and `num2`?
 ```js
 // your solution here
-```
+var num3 = num1 + num2
 6.  Use variables `num1`, `num2` and `num3` to fill in the `console.log()` to complete the sentence: 
 
 >The sum of 5 and 10 is 15
 
 ```js
-console.log()
+console.log(num1+num2=num3)
```

Comment:
the answer should be `console.log("The sum of " +num1+" and "+num2+ " is " + num3)`